### DISCUSSION TITLE
Don't write to `~/.icons/` for GTK cursor theme?

### DISCUSSION THREAD

**Discussion by isf63** [lxqt/lxqt#2585] [Q&A]
Currently the GTK cursor theme is written to 2 files:
```
~/.icons/default/index.theme
~/.config/gtk-3.0/settings.ini
```

`~/.icons/` is just one more hidden folder cluttering $HOME. It could be replaced with `~/.local/share/icons/` as that is also supported by both GTK2 and GTK3:

GTK3:
```
gtk/gtkicontheme.c
1290   self->search_path[i++] = g_build_filename (g_get_user_data_dir (), "icons", NULL);
1291   self->search_path[i++] = g_build_filename (g_get_home_dir (), ".icons", NULL);
```

GTK2:
```
gtk+-2.24.33/gtk/gtkicontheme.c
 578   priv->search_path[i++] = g_build_filename (g_get_home_dir (), ".icons", NULL);
 579   priv->search_path[i++] = g_build_filename (g_get_user_data_dir (), "icons", NULL);
```

**Comment by tsujan** [comment]
Like you, I don't like those dot folders, but that part of lxqt-config's code is already cluttered because of its nature; the removal of `~/.icons/` doesn't really make it cleaner. So, I'd say there's no harm in leaving it that way.
